content/_components/shopify-admin-v2/index.md

```diff
@@ -384,17 +386,17 @@ Examples for Object type `customers` are given below:
 
 #### Input Metadata
 
-* **First** - (strings, optional): Indicates amount of objects per execution. Default to 250
+* **First** - (strings, optional): Indicates amount of objects per execution. Default to 250.
   Input Metadata is depending on the configuration field `Number of search terms`.
   If `Number of search terms` is empty or equal `0`, additional fields will not be generated.
   If `Number of search terms` = 1, one search term will be added to metadata.
   If `Number of search terms` > 1, a number of search term equal `Number of search terms` and a number of criteria link equal 'Number of search terms - 1' will de created in metadata.
   Each search term has 3 fields:
-* **Field name** - chosen entity's field name. You need to select the one field from Allowed Values section
-  >**Please Note*:* Allowed Values section contains fields that are allowed to use in query and fields with `-` sign, which means `NOT` modifier (see [here](https://shopify.dev/docs/api/usage/search-syntax#modifiers))
+    - **Field name** - chosen entity's field name. You need to select the one field from Allowed Values section
+        >**Please Note:** Allowed Values section contains fields that are allowed to use in query and fields with `-` sign, which means `NOT` modifier (see [here](https://shopify.dev/docs/api/usage/search-syntax#modifiers)).
 
-* **Condition** - You need to select the one condition from Allowed Values section
-* **Field value** - the value that the field must match with the specified condition
+    - **Condition** - You need to select the one condition from Allowed Values section
```

Comment:
- **Condition** - You need to select one condition from the Allowed Values section